### ISSUE TITLE
Map Export

### ISSUE THREAD

**Issue by benni-tec** [flame-engine/tiled.dart#71]
# What could be improved
Exporting Maps from this package to tmx and json

# Why should this be improved
I will be creating an exporter (at least for xml/tmx) anyway, if you are interested in this functionality I would be happy to submit a PR to add this functionality to this package. If not I will create a seperate package for it 😄 

I have not implemented this yet, but will probably get to it next week. If you have any insights, recommendations or wishes I would appreciate a comment!

**Comment by spydon** [comment]
Sounds great!

**Comment by ufrshubham** [comment]
Yeah, this would be a nice feature to have. It'll will help in generating maps programmatically.
